react/components/RequestForm.tsx

```diff
@@ -706,4 +815,23 @@ class RequestForm extends Component<Props> {
   }
 }
 
-export default injectIntl(RequestForm)
+const compose = (...funcs) =>
+  funcs.reduce(
+    (a, b) =>
+      (...args) =>
+        a(b(...args)),
+    (arg) => arg
+  )
+
+export default compose(
+  graphql(NEAR_PICKUP_POINTS, {
+    options: () => ({
+      variables: {
+        lat,
+        long,
+      },
+      ssr: false,
+    }),
+    name: 'NearestPickupPoints',
+  })
```

Comment:
Instead of wrapping all in a HOC, let's start migrating to a function component. Please create a function component that wraps this one and pass down the props needed - the query and intl. We are doing something like that [here](https://github.com/vtex-apps/return-app/blob/aeaec1a9892a9bf63a12d61d2ae4c15cd316073d/react/StoreMyReturnsPageAdd.tsx#L24).